lib/reaction/tsx_handler.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Reaction
+  class TsxHandler
+    class << self
+      def render(path, assigns, context)
+        id = Pathname.new(path).relative_path_from(Rails.root.join("app", "views")).to_s.sub(".tsx", "")
+        response = Response.new(component: id, context:)
+        <<~HTML
+          <meta name="reaction-data" content="#{ERB::Util.url_encode(response.to_s)}" />
```

Comment:
This produces invalid HTML. According to the W3C, a `<meta>` element can not have a `name` attribute when it's in the `<body>`.

<img width="615" alt="image" src="https://github.com/user-attachments/assets/2133557b-9ab5-4c5b-972c-5a4f2c9898ba" />